### PR TITLE
[Solutions] Python解法 完善前三十题

### PR DESCRIPTION
**Pull request by 594Surefour**
Signed-off-by: paul_Lee <1337998606@qq.com>

---
name: PR
about: python code of first thirty questions
title: "[Solution/Add Question] 题目名称"
labels: status/pending
assignees: @594Surefour 

---

## 添加题目解法

* [相关链接（建议）]：Issue #154 
* [题目信息（必选）]：python codes of first thirty questions
* [题解思路（可选）]：

### CODE REVIEW

**Comment by QuakeWang** [comment]
Good job！

**Comment by QuakeWang** [comment]
@594Surefour 需要删除多余的文件: PAT/.DS_Store、PAT/cSrc/.DS_Store
以及修复以下问题：https://github.com/OS-EDU/KO--CSP/pull/227/checks?check_run_id=3755346827

File: PAT/pythonSrc/PAT-B1012.数字分类.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+n=input().split()
```
Comment (QuakeWang) [inline PAT/pythonSrc/PAT-B1012.数字分类.py:1]: 此文件未规范化

**Review by QuakeWang — COMMENTED** [review]
部分文件未格式化，建议进行格式化

**Comment by 594Surefour** [comment]
已更新！请查看

**Pull request closed without merging**